packages/amesos2/src/Amesos2_Superludist_TypeMap.hpp

```diff
@@ -70,6 +70,11 @@ namespace SLUD {
 
 extern "C" {
 
+  /// use the same function with name space in the macro
+#if defined(AMESOS2_ENABLE_SUPERLUDIST_5_4_0)
+#define USER_FREE(addr) SLUD::superlu_free_dist(addr)
```

Comment:
@kyungjoo-kim should this be `SUPERLU_FREE` as below? I don't see `USER_FREE` appearing anywhere else in the PR or in VOTD Amesos2.

Reply:
It is a macro used in  SUPERLU_FREE which is defined in superlu dist. 
```
/// this gives users can over-ride memory functions.
#if !defined(USER_FREE)
#define USER_FREE(addr) superlu_free_dist(addr)
#endif
#define SUPERLU_FREE(addr) USER_FREE(addr) 
```
As Amesos2 has namespace, I define the same default function with the name space.

Reply:
Thanks for clarification!